corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
i believe all the EDIT_DATA requires you to be superuser. i think this is the check that does it: https://github.com/benrudolph/commcare-hq/blob/forms-bulk-144444/corehq/apps/data_interfaces/dispatcher.py#L55